app/assets/javascripts/discourse/app/lib/user-menu/tab.js

```diff
@@ -0,0 +1,27 @@
+export default class UserMenuTab {
+  constructor(currentUser, siteSettings, site) {
+    this.currentUser = currentUser;
+    this.siteSettings = siteSettings;
+    this.site = site;
+  }
+
+  get shouldDisplay() {
+    return true;
+  }
+
+  get count() {
+    return 0;
+  }
```

Comment:
Do we need this function yet when it isn't being used?

Reply:
It's used here:

https://github.com/discourse/discourse/blob/3c60b97bd50dd5b4a0c3de1ff569d4e87f4ee5f3/app/assets/javascripts/discourse/app/components/user-menu/menu.hbs#L27-L29

Reply:
O icic. This was added in a previous commit. I was only searching through the current commit.